server/auth_server.go

```diff
@@ -39,11 +39,13 @@ func (a *AuthServer) IsWorkspaceAuthenticated(ctx context.Context, request *api.
 	}
 	workspaceAndNamespace := xOriginalAuth[0:pos]
 	pieces := strings.Split(workspaceAndNamespace, "--")
-	client := ctx.Value("kubeClient").(*v1.Client)
 	allowed, err := auth.IsAuthorized(client, pieces[1], "create", "apps/v1", "statefulsets", pieces[0])
```

Comment:
I believe we should be using `apps` for group here per our earlier discussion, please verify but making [this change](https://github.com/onepanelio/core/pull/174/files#diff-f9803e0eb312f8ff221f2cfd480173a3R65) worked for me.

Reply:
This appears to be working on my end as well.